simple_history/models.py

```diff
@@ -390,3 +390,44 @@ def __get__(self, instance, owner):
         values = (getattr(instance, f.attname)
                   for f in self.fields_included)
         return self.model(*values)
+
+
+class HistoricalChanges(object):
+    def diff_against(self, old_history):
+        if not isinstance(old_history, type(self)):
+            raise TypeError(("unsupported operand type(s) for diffing: "
+                             "'{}' and '{}'").format(
+                            type(self),
+                            type(old_history)))
+
+        model_delta_class = getattr(self, 'model_delta_class', ModelDelta)
```

Comment:
This is here so that we can potentially override the default `ModelDelta` with a different one? How would a user be able to set that?

Reply:
Looks like setting it would be set as an attribute `model_delta_class` on the class this is being mixed into. I don't see a compelling reason to support that so I'm happy to remove.